mycroft/client/speech/listener.py

```diff
@@ -141,15 +142,17 @@ def transcribe(self, audio):
             LOG.error("Could not request Speech Recognition {0}".format(e))
         except ConnectionError as e:
             LOG.error("Connection Error: {0}".format(e))
-            self.__speak("Mycroft seems not to be connected to the Internet")
+            self.__speak(mycroft.dialog.get("not connected to the internet",
+                                            self.stt.lang))
         except HTTPError as e:
             if e.response.status_code == 401:
                 text = "pair my device"
                 LOG.warn("Access Denied at mycroft.ai")
         except Exception as e:
             LOG.error(e)
             LOG.error("Speech Recognition could not understand audio")
-            self.__speak("Sorry, I didn't catch that")
+            self.__speak(mycroft.dialog.get("i didn't catch that",
+                                            self.stt.lang))
```

Comment:
There's an issue with the `stt.lang` variable in this case. It forces the part after the `-` to be upper case, for example `en-us` becomes `en-US` see `mycroft/sst/__init__.py` L44, This will not match the directory structure.

Alternatives:
- `dialog.get()` should use `lang.lower()` or convert before sending
- use `config_core.get('lang', 'en-us')` where `self.stt.lang` is used
- change the way the STT stores the lang variable (and do the upper case conversion when the value is used if needed)